pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.6.7</version>
+    <version>2.7.1</version>
```

Comment:
2.7.2 is out in the meantime. Unless there's a reason not to, we should use that.

Reply:
Unfortunate temporal overlap as it wasn't listed in the central repo when I did the switch last week 😅 
Has been adjusted.